demo/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,234 @@
+import { Story, Meta, html, Preview } from '@open-wc/demoing-storybook';
+
+<Meta title="Intro/Announcement" />
+
+# ING Bank open sources Lion
+## A library for building performant, accessible & flexible Web Components
+
+**TL;DR:** Web development is hard, wether you're making your own components, implementing Design Systems, incorporating accessibility, or adding third party dependencies. Lion aims to make your life a little bit easier for you, by taking the ground work of functional, accessible, and performant components out of your hands!
+
+As many of you may already know, ING has a long and rich history of Web Components, starting out with the Polymer library, all the way up to the recently released LitElement.
+
+[Need a refresher?](https://medium.com/ing-blog/ing-%EF%B8%8F-web-components-f52aacc71d7a) Web Components are a set of browser standards that allow us to write browser native, reusable, encapsulated, and modular components.
+
+Web Components standards have finally matured, and today, there are many ways to build and consume your Web Components, like: [Angular Elements](https://angular.io/guide/elements), [Stencil](https://stenciljs.com/docs/introduction/), [Vue](https://vuejsdevelopers.com/2018/05/21/vue-js-web-component/), [Svelte](https://svelte.dev/) and many, many more. And with the release of the new [Chromium Edge](https://www.theverge.com/2020/1/15/21066767/microsoft-edge-chromium-new-browser-windows-mac-download-os), this means that all major browsers support Web Components natively.
+
+So today, I'm happy to share with you all that ING is open-sourcing its own core library of Web Components: Lion! 🎉
+
+## Why Lion?
+
+Imagine the following fictional scenario:
+
+> Alfred works at Betatech, and is working on a new in-house application.
```

Comment:
Nice to have concrete example 👍 
Maybe make this example a bit more engaging by writing a few lines about Alfred, like a small persona?

Reply:
We actually purposely trimmed Alfred's section down by quite a bit because it got a very big story, we decided to make the 'story parts' to be short and 'snappy' instead

Reply:
Ah. Shouldn't become too big indeed, but mayne three lines instead of one. If I read this I think "who's Alfred? What is Betatech? What does Alfred do there and why?" But maybe that's just me ;)

Reply:
Ah. Shouldn't become too big indeed, but mayne three lines instead of one. If I read this I think "who's Alfred? What is Betatech? What does Alfred do there and why?" But maybe that's just me ;)